gensrc/proto/segment_v2.proto

```diff
@@ -151,6 +151,12 @@ message ColumnMetaPB {
     repeated ColumnIndexMetaPB indexes = 8;
     // pointer to dictionary page when using DICT_ENCODING
     optional PagePointerPB dict_page = 9;
+
+    repeated ColumnMetaPB children_columns = 10;
+    repeated string children_clolumn_names = 11;
+
+    // required
+    optional uint64 num_rows = 12;
```

Comment:
Add a comment about why we need this. In addition, perhaps num_values is better name?